contracts/src/token/erc20/mod.rs

```diff
@@ -825,7 +844,7 @@ mod tests {
             .setter(msg::sender())
             .set(one);
         let result = contract.transfer_from(Address::ZERO, alice, one);
-        assert!(matches!(result, Err(Error::InvalidSender(_))));
+        assert!(matches!(result, Err(Error::InvalidApprover(_))));
```

Comment:
Right now it looks weird with the test name. Maybe better test name?

Reply:
changed and replied in the other comment